tests/test_request.py

```diff
@@ -13,6 +13,7 @@ def setUp(self) -> None:
     def assert_method_calls_configure(self, method):
         with mock.patch("alma_client.credentials.ApiKeyCredentials.configure") as mocked_configure:
             getattr(self.request, method)()
+            _ = self.request.to_httpx()
```

Comment:
Since it's the `to_httpx()` which triggers the `configure()` call, the purpose of `assert_method_calls_configure()` is moot. We can assert that the `<method>()` call doesn't trigger the configure but that the `process_request()` does.